data/json/construction.json

```diff
@@ -819,6 +819,37 @@
     "pre_terrain": "t_pit_shallow",
     "post_terrain": "t_concrete"
   },
+  {
+    "type": "construction",
+    "description": "Fill Pit With Dirt",
+    "//": "Fills a deep pit with dirt.",
+    "category": "CONSTRUCT",
+    "time": 40,
+    "qualities": [ [ { "id": "SHOVEL", "level": 1 } ],
+    "components": [ [ "soil", 200 ] ],
+    "pre_terrain": "t_pit",
+    "post_terrain": "t_dirt"
+  },
+  {
+    "type": "construction",
+    "description": "Make Woodchip Floor",
+    "//": "Adds wood chippings to a shallowed dirt area to prevent plants growing, or for looks.",
+    "category": "CONSTRUCT",
+    "time": 40,
+    "components": [ [ "splintered_wood", 50 ] ],
+    "pre_terrain": "t_pit_shallow",
```

Comment:
You can probably put woodchips down on any dirt area

Reply:
You can. There's usually two kinds, one is a dug box to add depth, and then there's an area just sprinkled with them. In this case I'm fine with it being the standard chip path.